src/service/action-impl.js

```diff
@@ -737,6 +779,23 @@ class ParserTokenizer {
       return {type: TokenType.LITERAL, value, index: newIndex};
     }
 
+    // Object literal.
+    if (c == OBJECT_SET[0]) { // '{'
+      let end = -1;
+      for (let i = newIndex + 1; i < this.str_.length; i++) {
+        if (this.str_[i] == OBJECT_SET[1]) { // '}'
+          end = i;
+          break;
+        }
+      }
+      if (end == -1) {
+        return {type: TokenType.INVALID, index: newIndex};
+      }
+      const value = this.str_.substring(newIndex, end + 1);
+      newIndex = end;
+      return {type: TokenType.LITERAL, value, index: newIndex};
```

Comment:
Thought about that too but technically this is still a "literal". Anyways, done.